tests/Parse/ParseSchemaTest.php

```diff
@@ -164,6 +164,23 @@ public function testUpdateSchemaStream()
         $this->assertNotNull($result['fields']['status']);
     }
 
+    public function testUpdateMultipleSchemaStream()
+    {
+        ParseClient::setHttpClient(new ParseStreamHttpClient());
+
+        $schema = self::$schema;
+        $schema->save();
+
+        $schema->addString('name');
+        $schema->update();
+        $schema->update();
```

Comment:
Double update?

Reply:
There was a bug where calling multiple updates would create multiple fields / indexes. I don't think this was intended functionality. Thus the `testUpdateMultipleSchemaStream`

Reply:
Ahh, this is fine then.